client.go

```diff
@@ -23,6 +23,11 @@ type gocloak struct {
 	restyClient *resty.Client
 	Config      struct {
 		CertsInvalidateTime time.Duration
+		authAdminRealms     string
```

Comment:
1.  Let's move this structure to outside of gocloak and the GoCloak interface needs a function like Config() that will return a pointer for the instance of Config.
2. All the Fields should be exported, otherwise they are not available in external packages.